website/source/docs/secrets/ssh/signed-ssh-certificates.html.md

```diff
@@ -12,195 +12,326 @@ description: |-
+team, or configuration management tooling.
 
-### Configure the host CA certificate
+1. Mount the backend. Like all secret backends in Vault, the SSH secret backend
+must be mounted before use.
 
-```text
-vault write -f ssh-host-signer/config/ca
-Key             Value
----             -----
-public_key      ssh-rsa
-AAAAB3NzaC1yc2EAAAADAQABAAACAQDB7lbQ4ub8/7AbfnF4zntCxPGijXzdo50MrUz0I1Vz/n98pBPyrUEcNqFeCr6Qs9kRL8J8Bu48W/DtSxAiScIIlzyezulH+wxdZnCN3sxDUhFbTUosWstSt6P5BZbJsMTpfHfuYbOexewX9ljw636fUrRtU3/0Emq2FkOcKEv9v7z+Qwdaom+8R6CvRA0Kit0yyh6DXD+IjPnOmhYFoNH6ZhPU2xJ+7M16vLpCmH6OO1krHlDK+ZUvVKD2ZjfmbegoYEV/9FETSzVnQ/pOrP0SdwpmGhei7VO7hQwa6zOWIah66TYYdaIopHFha60rZPybliZFAi/CcUO7NYldq7Q2ty68RKp+Im2ROPbXLKYc2N+PThWwbxA1yQ7shvgSy/JEbAcYQuDLGn6tA6n5sXpvFcJe+vlf75w+laXIPUXTVHr2g1O5Irzwunpg03Pa+Kab8Hp8uq61XfTfNtCwHYr/K5YO27rHcpMA/gBKkOFmBbcI3VlUbi266HiyFANz0JMObzXS5hEGm90NevMD2nEFvZHYHx4hXRtoox7Bh4CCj2fUPaThnuTvv1hLBgryWxwDXL7OwHGx6wd/dpXo/1fG1DRUotfVcJP7k7FG9MUAB6tOR2RCuXEaTF6bv3tCpbsdzzzyhs/aIa6IeYRV4HUo3nW5g/IRT+6F/MLd63Cq5w==
-```
+    ```text
+    $ vault mount -path=ssh-client-signer ssh
+    Successfully mounted 'ssh' at 'ssh-client-signer'!
+    ```
 
-The returned host CA public key should be added to `known_hosts` file in all
-client machines with a `@cert-authority <domain>` prefix, where `<domain>` can
-be any value (single host, wildcard domain, wildcard) SSH accepts to define
-hosts (e.g. `@cert-authority *.example.com ssh-rsa AAAA...`). This public key
-can also be retrieved using `vault read ssh-host-signer/config/ca`.
+    This mounts the SSH backend at the path "ssh-client-signer". It is possible
+    to mount the same secret backend multiple times using different `-path`
+    arguments. The name "ssh-client-signer" is not special - it can be any name,
+    but this documentation will assume "ssh-client-signer".
 
-### Configure the client CA certificate
+1. Configure Vault with a CA for signing client keys using the `/config/ca`
+endpoint. If you do not have an internal CA, Vault can generate a keypair for
+you.
 
-```text
-vault write -f ssh-client-signer/config/ca
-Key             Value
----             -----
-public_key      ssh-rsa
-AAAAB3NzaC1yc2EAAAADAQABAAACAQDoYiTm0QrK+NGkv8H/Sq1Q9l2PQwxY1ROSPpRqOoFAgCM7vtEZSGchm8j3/Da6Vj84sUg+09N+dwMZ6z3HG57dBWyfHGCbEt+fIJtRcCV7KkQ4xktBdapq5PychBg0bYIP0+JWXe7yOAR6s+DWJgUUHwmYiY0GlrzoztvtYcU/pvj4zWY/EWcCAYRiZThqJOJ9rDzhEzmVHVYelZ8dlehKZsUOQ3dY5fCHgWJ5/IKSfGUI1w1h/3ENAToBKInbW2J/Wy0mbsfUJhha84zgqSeHd4gZ9YqaSpzYffVvDz6Ud+KtPWt4ejJDyvJVFAVnVIvVUCkuXVefBlC10Ww6l52DXex/rdxQW+46ZNV5zlFOUi4VWFPT6bOlIEj1hgHG0cfFKARQXcVlp03xTi5DRJp3jlBf/QLeUy4kZEFUdQl+gkVLwySrOh+1kHwE15JQWpNHxfxWzYnNnjokT2RCW2ANO4Y8vfMPCVM5OKSVNISHm3UurNkJNTAomK/ElnXQgP23UcVgbF3mNRBPx9U3/+9HHkXcqDefSI7XBC7npIBFdetCxFt3nW/cX4l1zfq6ZExCVde+/40V1U+TOoLZVLIZZzEuA40J4H99oK7QN5/MtCGaSr7kgAx+7ymfSY37rXB6wOVi5HTBlhyh+VFGU+Wmy+q5sxfr6+ZLGwoGpvNh0Q==
-```
+    ```text
+    $ vault write ssh-client-signer/config/ca generate_signing_key=true
```

Comment:
The docs used to say it's false by default, but that appears to have changed 😄 . I'm still in favor of making it explicit, since otherwise new users might be confused about "what is the `-f` in here for". A few people asked why we pass "force" into the command.